frontend/test/metabase/scenarios/models/reproductions/22715-remapped-values-override-column-identifier.cy.spec.js

```diff
@@ -26,10 +26,11 @@ describe.skip("filtering based on the remapped column name should result in a co
 
       // Let's go straight to the model metadata editor
       cy.visit(`/model/${id}/metadata`);
-      // Without this Cypress fails to remap the column because an element becomes detached from the DOM
-      cy.findByText(
-        "Use the tab key to navigate through settings and columns.",
-      );
+
+      // Without this Cypress fails to remap the column because an element becomes detached from the DOM.
+      // This is caused by the DatasetFieldMetadataSidebar component rerendering mulitple times.
+      cy.findByText("Database column this maps to");
+      cy.wait(3000);
```

Comment:
@nemanjaglumac -- I know this is incredibly hacky, but I couldn't think of an alternative. Is this fine?

The component this selector targets rerenders an inordinate amount so we need to wait a long while for it to settle down. You can see here that it takes seconds for it to stop rerendering... not good. I will try to fix this soon.

https://user-images.githubusercontent.com/13057258/184419208-fbe2675e-5480-42ca-ad97-dd2817c0e96d.mov

Reply:
🤢

Reply:
> I will try to fix this soon.

@daltojohnso 
Did you mean in this PR or soon as in - after this gets merged?

Reply:
Just spent couple hours trying out different approaches but literary nothing seems to work. The way this component behaves is an anomaly which should be fixed, rather than trying to find a workaround for it in Cypress.

Sorry I couldn't help.

Reply:
I meant after this merged. The problem component is just too complex. fyi -- I'm abandoning this PR and moving to https://github.com/metabase/metabase/pull/24817